### ISSUE TITLE
[Feature] FBX Exporter with Vertex Colors and Vertex Alpha Support

### ISSUE THREAD

**Issue by Lilothestitch16** [hogsy/jaded#18]
Would it be possible to add a feature to export every .gao file as .fbx with vertex colors and vertex alpha?

**Comment by hogsy** [comment]
I'll be looking to support this via [glTF](https://en.wikipedia.org/wiki/GlTF) probably rather than FBX. Will leave this open for now.

**Comment by Escope12** [comment]
What’s glTF? Does it support bones and vertex colors?

**Comment by hogsy** [comment]
I did link you to a page there covering it, but yes, it supports everything FBX supports and more. Every modern modelling tool (including pretty much every modern engine, including UE4/5 + Godot) will support glTF, which is an open standard, whereas FBX is proprietary and doesn't really have much of a future.

**Comment by Escope12** [comment]
Perfect. That should work.

**Comment by Escope12** [comment]
Also will there be support for exporting the character models as well?

**Comment by hogsy** [comment]
That's the plan™

**Comment by Escope12** [comment]
I look forward to that. Also will there be an option to export textures as .png? And will there be support for Beyond Good & Evil HD Remastered?

**Comment by hogsy** [comment]
> Also will there be an option to export textures as .png?

Nope, textures will be in TGA format - you can do the conversion yourself pretty easily if necessary.

**Comment by Escope12** [comment]
And support for Beyond Good & Evil HD?

**Comment by hogsy** [comment]
That's a bit beyond the scope of this and not something I'm currently planning for :p 
The content Droolie has pulled over from BGE and packed into a BF will be exportable when these changes are in, but beyond that, no, not at this time.

**Comment by Escope12** [comment]
How did Droolie pull the data over from BGE?

**Comment by hogsy** [comment]
I'm not going to have a whole conversation with ya via GitHub lmfao, feel free to DM me on Discord (hogsy#9210) or whatever, keeping in mind I'm going to bed, but I think we're getting well away from the subject of this ticket :p 

The gist is that he modified [ray1map](https://github.com/BinarySerializer/Ray1Map) to support Jade's formats and then exported from there to a new BF in such a way that the editor is able to load the content. If you want to know more, I recommend messaging Droolie directly.

**Comment by Escope12** [comment]
Okay I’ll ask him about it on Discord. Sorry to go off topic.

**Comment by hogsy** [comment]
Going to close this one as it feels like a duplicate of [this](https://github.com/hogsy/jaded/issues/4).

**Comment by Lilothestitch16** [comment]
That's fine. I'm okay with this being closed.
